packages/docs/stories/components/range.stories.ts

```diff
@@ -0,0 +1,453 @@
+      description: {
+        story: generateStoryDescription('range', 'custom-track-colors'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-range
+      class="custom-track-color"
+      max="100"
+      min="0"
+      value="50"
+    ></syn-range>
+    <style>
+      .custom-track-color {
+        --track-color-active: var(--syn-color-success-600);
+      }
+    </style>
+  `,
+};
+
+export const CustomTrackOffset: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('range', 'custom-track-offset'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-range
+      class="custom-track-offset"
+      max="100"
+      min="0"
+      value="25"
+    ></syn-range>
+    <style>
+      .custom-track-offset {
+        --track-active-offset: 50%;
+      }
+    </style>
+  `,
+};
+
+export const MultiKnob: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('range', 'multi-knob'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-range max="100" min="0" value="30 70"></syn-range>
+  `,
+};
+
+export const MultiKnobWithRestrictedMovement = {
+  parameters: {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    docs: {
+      description: {
+        story: generateStoryDescription('range', 'multi-knob-restrict-movement'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-range
+      class="range-restrict-movement"
+      value="30 70"
+      label="Demo of restricting values"
+      min="0"
+      max="100"
+      step="1"
+    ></syn-range>
+    <script type="module">
+      Array
+        .from(document.querySelectorAll('.range-restrict-movement'))
+        .forEach((el) => {
+          el.addEventListener('syn-move', e => {
+            const { detail, target } = e;
+            const values = target.valueAsArray;
+            const { knob, value } = detail;
+
+            if (knob === target.knobs[0] && value > values[1]) {
+              e.preventDefault();
+            }
+
+            if (knob === target.knobs[1] && value < values[0]) {
+              e.preventDefault();
+            }
+          });
+        });
+    </script>
+  `,
+};
+
+export const Ticks: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('range', 'ticks'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-range
+      class="syn-range-with-tick"
+      max="100"
+      min="0"
+      value="50"
+      label="Volume"
+    >
+      <nav slot="ticks">
+        <syn-range-tick label="0"></syn-range-tick>
+        <syn-range-tick label="50"></syn-range-tick>
+        <syn-range-tick label="100"></syn-range-tick>
+      </nav>
+    </syn-range>
+    <style>
+      .syn-range-with-tick nav {
+        justify-content: space-between;
+        flex-direction: row;
+        display: flex;
+      }
+    </style>
+  `,
+};
+
+export const TooltipPlacement: Story = {
```

Comment:
Doesn't show correct position in docs.

![CleanShot 2024-07-16 at 10 32 20](https://github.com/user-attachments/assets/8f24c707-4d6a-4f58-b497-57cebfc22bc6)

Reply:
Fixed